todo.py

```diff
@@ -1,21 +1,62 @@
 
-def print_todos(todos):
-    if len(todos) == 0:
+pending_list = []
+completed_list = []
+
+def print_todos(pending_list, completed_list):
+    pending_menu_top = """
+====== Pending ======
+    """
+    print(pending_menu_top)
+    if len(pending_list) == 0:
         print("You have nothing to do.")
     else:
         i = 0
-        for todo in todos:
+        for todo in pending_list:
             print(f"{i}: {todo}")
             i += 1
+    pending_menu_bottom = """
+=====================    
+    """
+    print(pending_menu_bottom)
+    
+    completed_menu_top = """
+====== Completed ======
+    """
+    print(completed_menu_top)
+
+    i = 0
+    for todos in completed_list:
+        print(f"{i}: {todo}")
+        i += 1
+
+    completed_menu_bottom = """
+=======================    
+    """
+    print(completed_menu_bottom)
+
+def add_todo(lst, item):
+    new_dict = {}
+    new_dict[item] = 'do the thing'
+    new_dict['completed'] = False
+    
+    lst.append(new_dict)
+    
 
-def add_todo(todos, item):
-    todos.append(item)
+def complete_todo(todos, index):
```

Comment:
Since you're using two lists to manage the different states of the todos, I wonder if `complete_todo` should be updated to accept three arguments? (pending todos, completed todos, and the index to update)